### PR TITLE
Fix 

### PR DESCRIPTION
**Pull request by v-nakon**
fixed html and css file. js will fix next time.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
+	<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
 	<link rel="stylesheet" type="text/css" href="style.css">
-	<style>
-	
-  
-	</style>
   </head>
-  <body>
+  <header>
+    <div class="header">
```
Comment (byzyk) [inline index.html]: why do you need another `div` here? You already got `header`

File: index.html [hunk 2]
```diff
@@ -2,21 +2,23 @@
...
+    <div class="logo">LOGO</div>
     <nav class="four">
       <ul>
-        <li class="logo"><p>LOGO</p></li>
+        <!--<li class="logo"><p>LOGO</p></li>-->
```
Comment (byzyk) [inline index.html]: I guess this is not needed

File: index.html [hunk 3]
```diff
@@ -2,21 +2,23 @@
...
         <li><a href="#">Home</a></li>
         <li><a href="about.html">About</a></li>
       </ul>
     </nav>
+	</div>
+  </header>
+	<body>
```
Comment (byzyk) [inline index.html]: `<body>` is in the wrong place. See the doc example here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body#Example

File: style.css [hunk 4]
```diff
@@ -1,5 +1,5 @@
 body {
-  margin:1;
+  margin: auto;
   width: 600px;
```
Comment (byzyk) [inline style.css]: It's not a good practice to style `body` tag like this. If you need to apply these styles it's better to create a new container inside `body`, something like: `<div class="container">` and then style `.container` class.

File: style.css [hunk 5]
```diff
@@ -26,13 +26,14 @@ a {
...
 a:hover {background: white}
 
-p {
-   font-family: Lobster;
-   padding-left: 1em;
-   padding-right: 1em;
-   }
 .logo {
   background: white;
+  font-family: 'Lobster';
```
Comment (byzyk) [inline style.css]: font fallback is missing

File: style.css [hunk 6]
```diff
@@ -26,13 +26,14 @@ a {
...
+  float:left;
+  padding-left: 1em;
+  padding-right: 1em;
+  padding-top: 1em;
+  padding-bottom: 0.9em;
```
Comment (byzyk) [inline style.css]: paddings can be combined into single line declaration like this: `padding: <top> <right> <bottom> <left>`. See the doc: https://developer.mozilla.org/en-US/docs/Web/CSS/padding

File: index.html [hunk 7]
```diff
@@ -2,31 +2,31 @@
...
       <ul>
-        <li class="logo"><p>LOGO</p></li>
         <li><a href="#">Home</a></li>
         <li><a href="about.html">About</a></li>
       </ul>
     </nav>
-    <h1>My web page!</h1>
+	</div>
+  </header>
+  <body>
```
Comment (byzyk) [inline index.html]: Everything on the webpage should be inside `body` tag. You need to place `header` inside your container, in this case, you also wouldn't have to duplicate CSS for both `container` and `head` classes

File: style.css [hunk 8]
```diff
@@ -26,13 +26,11 @@ a {
...
 a:hover {background: white}
 
-p {
-   font-family: Lobster;
-   padding-left: 1em;
-   padding-right: 1em;
-   }
 .logo {
   background: white;
+  font-family: 'Lobster', 'Arial';
```
Comment (byzyk) [inline style.css]: 'Arial' might not be available on some OS (for example on macOS). It's also considered as good practice to include `sans-serif` in the end.

File: index.html [hunk 9]
```diff
@@ -2,31 +2,31 @@
...
-      <li class="slide">Slide 3</li>
-    </ul>
-	<script src="slider.js"></script>
-
+      <ul id="slides">
+        <li class="slide showing">Slide 1</li>
+        <li class="slide">Slide 2</li>
+        <li class="slide">Slide 3</li>
+      </ul>
+	  <script src="slider.js"></script>
```
Comment (byzyk) [inline index.html]: `script` tags should be in the very bottom of the page before closing `body` tag. Not inside any `div` tags.

Comment (v-nakon) [reply index.html]: can include <script>  in <div class="container">?

File: index.html [hunk 10]
```diff
@@ -5,28 +5,26 @@
...
+        <div class="logo">LOGO</div>
+          <nav class="four">
+            <ul>
+              <li><a href="#">Home</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </nav>
+	  </div>
+    </header>
+    <body>
```
Comment (byzyk) [inline index.html]: None of HTML view tags (such as `div`, `header`, `ul` etc) can be outside of `body` tag.

File: index.html (hunk 9 above)
Comment (byzyk) [reply index.html]: You can include it anywhere but it's better to place it in the very bottom before closing `body` tag.

File: style.css [hunk 11]
```diff
@@ -28,7 +28,7 @@ a:hover {background: white}
 
 .logo {
   background: white;
-  font-family: 'Lobster', 'Arial';
+  font-family: 'Lobster', sans-serif;
```
Comment (byzyk) [inline style.css]: 👍🏽

**Pull request merged**